checker/build.xml

```diff
@@ -1508,6 +1508,8 @@
          <env key="LT_BIN" value="${LT_BIN}"/>
       </exec>
       <move file="jdk/jdk.jar" tofile="jdk/${jdkName}"/>
+        <!-- Copy jars to dist. If dist doesn't exist then the copy task creates it.-->
+        <copy file="jdk/${jdkName}" tofile="dist/${jdkName}"/>
```

Comment:
This should have `overwrite="true"` and maybe also `failonerror="false"` to account for an already existing file.

Reply:
By default, if the file exists and is older than the copied file, it is overwritten.  Using `overwrite="true"`  overwrites existing files if they are newer than the copy.  I don't think we want that.

Reply:
It's what `dist-nobuildjdk` was doing. Fine with me if it works without that option.